pages/common/dolt-version.md

```diff
@@ -0,0 +1,8 @@
+# dolt version
+
+> Displays the current Dolt cli version.
```

Comment:
```suggestion
> Displays the current dolt CLI version.
```

Reply:
solved in 74cec9a7f6adbce48e3678548f1f10b58aad33eb